File: C++/Lessons/CW-04-12-2025/003.cpp
#include <iostream>

void print(int* arr, int size);

int main()
{
    const int size{10};
    int arr[size]{1,4,2,6,34,87,223,6,23,4};

    print(arr, size);

    return 0;
}

void print(int* arr, int size)
{
    for(int i{}; i < size; i++)
    {
        std::cout << *(arr + i) << " ";
    }
    std::cout << "\n";
}

File: C#/Homework/03-03/003.cs
public class HW
{
    public int size;
    public int[] arr = new int[size];
    public HW(int s)
    {
        size = s;
    }

    public compress()
    {
        int no_zero_index = 0;
        for (int i = 0; i < arr.Length; i++)
            if (arr[i] != 0) no_zero_index = i;
        
        
    }
}



